httpclient/build.gradle.kts

```diff
@@ -31,6 +33,13 @@ dependencies {
 
 tasks.test {
   useJUnitPlatform()
+  testLogging {
+    events("passed", "skipped", "failed", "standardOut", "standardError")
+    exceptionFormat = TestExceptionFormat.FULL
+    showExceptions = true
+    showCauses = true
+    showStackTraces = true
```

Comment:
Why wouldn't this be the default behavior w/ gradle?

Reply:
shall we ask [Hans Dockter](https://www.linkedin.com/in/hansdockter/)?

Reply:
I got questions and I'm not leaving without answers